packages/box/src/Box.story.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, select } from '@storybook/addon-knobs';
+import Box from '.';
+
+storiesOf('Box', module)
+  .add('Default', () => <Box>By default I am a div</Box>)
```

Comment:
I think it would be better to use knobs for demonstrating changing props.

Reply:
👍

Reply:
Discussed offline, leaving as is